### PR TITLE
Sync 3.10 with develop

### PR DESCRIPTION
**Pull request by SaikrishnaBairamoni**
# PR Details
This PR syncs the 3.10.0 release back into develop with Docker files points to develop.
## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: ford_fusion_sehybrid_2019/docker-compose.yml [hunk 1]
```diff
@@ -108,6 +108,7 @@ services:
 
   velodyne_lidar_driver:
     image: usdotfhwastoldev/carma-velodyne-lidar-driver:develop
+    image: usdotfhwastol/carma-velodyne-lidar-driver:develop
```
Comment (msmcconnell) [inline ford_fusion_sehybrid_2019/docker-compose.yml:111]: This duplicated line needs to be removed. (It also has the wrong organization name)

File: freightliner_cascadia_2012_dot_10002/carma_docker.launch [hunk 2]
```diff
@@ -50,7 +50,10 @@
   <!-- List of String: Guidance Control Plugins that will be validated by the Guidance Plugin Validator Node if enabled -->
   <arg name="control_plugins_to_validate" value="[]"/>
   
+<<<<<<< HEAD
 
+=======
+>>>>>>> master
```
Comment (msmcconnell) [inline freightliner_cascadia_2012_dot_10002/carma_docker.launch:56]: Invalid merge

File: lexus_rx_450h_2019/carma_docker.launch [hunk 3]
```diff
@@ -49,7 +49,11 @@
   <arg name="tactical_plugins_to_validate" value="[]"/>
   <!-- List of String: Guidance Control Plugins that will be validated by the Guidance Plugin Validator Node if enabled -->
   <arg name="control_plugins_to_validate" value="[]"/>
+<<<<<<< HEAD
 
+=======
+  
+>>>>>>> master
```
Comment (msmcconnell) [inline lexus_rx_450h_2019/carma_docker.launch:56]: Invalid merge

**Review by msmcconnell — CHANGES_REQUESTED** [review]

File: ford_fusion_sehybrid_2019/docker-compose.yml (hunk 1 above)
Comment (SaikrishnaBairamoni) [reply ford_fusion_sehybrid_2019/docker-compose.yml]: Removed Duplicate lines and updated to correct Organization name

File: freightliner_cascadia_2012_dot_10002/carma_docker.launch (hunk 2 above)
Comment (SaikrishnaBairamoni) [reply freightliner_cascadia_2012_dot_10002/carma_docker.launch]: resolved

File: lexus_rx_450h_2019/carma_docker.launch (hunk 3 above)
Comment (SaikrishnaBairamoni) [reply lexus_rx_450h_2019/carma_docker.launch]: resolved

**Review by msmcconnell — APPROVED** [review]

**Pull request merged**